tools/static-assets/skel/.meteor/packages

```diff
@@ -19,3 +19,4 @@ ecmascript              # Enable ECMAScript2015+ syntax in app code
 
 autopublish             # Publish all data to the clients (for prototyping)
 insecure                # Allow all DB writes from clients (for prototyping)
+census                  # Track anonymous info to help us improve Meteor
```

Comment:
Truly agree with @mquandalle . Even MDG wants this package to collect data, I suggest not adding it as a default. I think, it's not bad to add it to the skeleton app, but you should notify the user once the app is created. Also, you should notify users by print some logs like `Census is added, Meteor will use this package to collect data. You can remove it by $ meteor remove census.`
I'm against adding it in a update.